src/index.js

```diff
@@ -0,0 +1,140 @@
+// `"use strict";`
+
+const state = {
+    increaseTempButton: null,
+    decreaseTempButton: null,
+    tempNumberContainer: null,
+    tempNumberClass: '',
+    tempNumber: '',
+    skyEmojiContainer: null,
+    skyEmoji: '',
+    landEmojiContainer: '',
+    landEmoji: '',
+    cityNameContainer: null,
+    cityName: '',
+    cityInput: '',
+    realTempButton: null,    
+    SkyDropdown: null,
+    lat: null,
+    lon: null,
+};
+
+const loadControls = () => {
+    state.increaseTempButton = document.getElementById('increaseTempButton');
+    state.decreaseTempButton = document.getElementById('decreaseTempButton');
+    state.tempNumber = parseInt(document.getElementById('tempNumberContainer').innerText);
+    state.tempNumberContainer = document.getElementById('tempNumberContainer');
+    state.tempNumberClass = document.getElementById('tempNumberContainer').className;
+    state.skyEmojiContainer = document.getElementById('skyEmojiContainer');
+    state.skyEmoji = document.getElementById('skyEmojiContainer').innerText;
+    state.landEmojiContainer = document.getElementById('landEmojiContainer');
+    state.landEmoji = document.getElementById('landEmojiContainer').innerText;
+    state.cityNameContainer = document.getElementById('cityNameContainer');
+    state.cityName = document.getElementById('cityNameContainer').innerText;
+    state.cityInput = document.getElementById('cityInput');
+    state.realTempButton = document.getElementById('realTempButton');
+    state.skyDropdown = document.getElementById('skyDropdown');
+    state.weatherCity = document.getElementById('weatherCity');
+};
```

Comment:
Oh wow! I love this! This shows me you centering readability! Naming conventions like this make your code a lot more intuitive!